Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -732,6 +729,19 @@ public bool DoHarvest(EntityUid plantholder, EntityUid user, PlantHolderComponen
         return true;
     }
 
+    /// <summary>
+    /// Force do scream on PlantHolder (like plant is screaming) using seed's ScreamSound specifier (collection or soundPath)
+    /// </summary>
+    /// <returns></returns>
+    public bool DoScream(EntityUid plantholder, SeedData? seed = null)
+    {
```

Comment:
i feel like this would be better if you simply added the check for `CanScream` in this function and then removed the check from all the callers.

Reply:
ok